libraries/abstractions/secure_sockets/test/iot_test_tcp.c

```diff
@@ -2961,6 +2962,61 @@ TEST( Full_TCP, AFQP_SOCKETS_htons_HappyCase )
 }
 /*-----------------------------------------------------------*/
 
+TEST( Full_TCP, test_dns_cache_multiple_addresses )
+{
+    BaseType_t xResult = pdFAIL;
+    uint32_t i;
+    uint32_t j;
+    uint32_t ulIPAddress;
+    uint32_t ulUnique;
+    uint32_t ulNumUniqueIPAddresses = 0;
+
+    /* Resolve the AWS IoT broker endpoint, which will have multiple IP addresses */
+
+    uint32_t ulIPAddresses[ 10 ] = { 0UL };
+
+    tcptestPRINTF( ( "Starting %s.\r\n", __FUNCTION__ ) );
+    /*
+     * Resolve the broker endpoint to an array of IP addresses. Each
+     * subsequent call will return one of the addresses which the
+     * name resolves to.  Call once more than the maximum number of addresses
```

Comment:
>>  Call once more than the maximum number of addresses per cache entry to ensure that wraparound works.

How is this ensured?

Reply:
This comment is misleading for a couple of reasons, I will change it.  First, since this is Secure Sockets API-level test, it has no idea what sort of DNS caching strategy is employed (if any as pointed out in a previous comment), how many addresses could be stored per entry, etc.  The only thing it needs to ensure is that subsequent calls to `SOCKETS_GetHostByName()` on an AWS IoT endpoint return different IP addresses.

Wraparound testing in the FreeRTOS+TCP implementation of `FreeRTOS_gethostbyname()` needs to be handled by a unit test in that code, not a system-level test here.